README.md

```diff
@@ -8,21 +8,21 @@ applications and application instances.
 
 Add log_weasel to your Gemfile:
 
-<pre>
```

Comment:
Thank you for cleaning up the formatting in this file. Love having FE expertise on the team! 😂